lambda/write-user-services/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "write-user-services",
+  "version": "1.0.0",
+  "description": "A Lambda function to write events to DynamoDB",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```

Comment:
Could do `GDS` or `GDS - One Login for Government`?

Reply:
Let's go with `GDS` - I'm still not sure what our directorate / programme / product are called 😂